task/fbc-related-image-check/0.1/fbc-related-image-check.yaml

```diff
@@ -28,8 +28,10 @@ spec:
           cpu: 10m
       script: |
         #!/usr/bin/env bash
-        set -o pipefail
+        set -euo pipefail
         source /utils.sh
+        trap 'handle_error $(results.TEST_OUTPUT.path)' EXIT
+
         FAILEDIMAGES=""
         catalog="$(find $(workspaces.workspace.path)/hacbs/fbc-validation/ -name catalog.yaml)"
 
```

Comment:
I cannot comment there, but in line 40, 42; this will stop work
```
        relImgs="$(yq -r '.relatedImages[]?.image' $catalog)"
        if [ $? -ne 0 ]; then
```
shuold be changed to
``` 
        status=0
        relImgs="$(yq -r '.relatedImages[]?.image' $catalog || staus=$?)"
        if [ $? -ne $status ]; then
```

Reply:
EDIT: `||` must be outside of `$()`

Reply:
I am thinking, maybe we should remove `set -euo pipefail` since we will have that `trap EXIT`.

Reply:
without `euo pipefail` faulty code won't trigger exit, so having trap EXIT without this doesn't do much

Reply:
Fixed. Thanks.